README.md

```diff
@@ -1,3 +1,47 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/thjw94949tm5lbw5/branch/master?svg=true)](https://ci.appveyor.com/project/nvborisenko/client-net/branch/master)
-[![NuGet Badge](https://buildstats.info/nuget/reportportal.client)](https://www.nuget.org/packages/reportportal.client)
-[![Coverage](https://codecov.io/gh/reportportal/client-net/branch/master/graph/badge.svg)](https://codecov.io/gh/reportportal/client-net)
+[![Build status](https://ci.appveyor.com/api/projects/status/thjw94949tm5lbw5/branch/master?svg=true)](https://ci.appveyor.com/project/nvborisenko/client-net/branch/master) [![NuGet Badge](https://buildstats.info/nuget/reportportal.client)](https://www.nuget.org/packages/reportportal.client) [![Coverage](https://codecov.io/gh/reportportal/client-net/branch/master/graph/badge.svg)](https://codecov.io/gh/reportportal/client-net)
+
+# Project Name
+API client for https://reportportal.io
+
+## General Information
+ This is a library containing methods, which allow to send the calls directly to Report portal and uses the RP API.
+
+## Technologies Used
+.NET Core, System.Net.Http
+
+## Features
+List the ready features here:
+- Start, update, finish, delete launch;
+- Create logs with all levels;
+- Create/update user filter;
+
+## Setup
+Download using Nuget package console:
+
+PM> Install-Package ReportPortal.Client
+
+## Usage
+Example for starting launch in Report Portal:
+
+````C#
+//Create service object
+//Uuid value is specific for exact user, it could be found in Report Portal: User profile -> Access token
+ var service = new ReportPortal.Client.Service(new Uri("https://rp.epam.com/api/v1/"), "ProjectName", "uuid");
```

Comment:
I suggest to use [demo reportportal endpoint](https://demo.reportportal.com/api/v1) here.

Reply:
Done